<launch>

    <param name="use_sim_time" value="true" />

    <node pkg="rviz" type="rviz" name="display_result" output="screen" args="-d $(find localization)/config/check_path.rviz" />

    <node pkg="localization" type="CheckMsgPath" name="CheckMsgPath" output="screen">
        <param name="init_x" type="double" value="1716.40466"/>
        <param name="init_y" type="double" value="1014.501642"/>
        <param name="init_yaw" type="double" value="-2.18"/>
        <param name="map_path" type="string" value="/home/gogochiou/SDC_ws/data/nuscenes_map.pcd" />
    </node>

</launch>